tools/export_user/export_user.py

```diff
@@ -0,0 +1,73 @@
+import argparse
+import logging
+import os
+import sys
+reload(sys)
+sys.setdefaultencoding('utf-8')
+import json
+import csv
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), '..', '..'))
+from safetypy import safetypy as sp
+from collections import OrderedDict
+
+# Possible values here are DEBUG, INFO, WARN, ERROR and CRITICAL
+LOG_LEVEL = logging.DEBUG
+
+DEFAULT_CONFIG_FILENAME = 'config.yaml'
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-t', '--token', required=True)
+    args = parser.parse_args()
+    api_token = args.token
+    sc_client = sp.SafetyCulture(api_token)
+    my_groups_and_orgs = json.loads(sc_client.get_my_groups().content)
+    org_id = ""
+    groups_list = []
+    user_map={}
+
+    for i in my_groups_and_orgs['groups']:
+        if i['type'] == 'organisation':
+            org_id = i['id']
+            users_of_org = json.loads(sc_client.get_users_of_groups(org_id))
+            for i in users_of_org['users']:
+                if i['status'] == 'active':
+                    email = i['email']
+                    user_map[email] = {'groups': [], 'firstname': i['firstname'], 'lastname': i['lastname'] }
+
+    json_all_groups = json.loads(sc_client.get_all_groups_in_org().content)
+    for group in json_all_groups['groups']:
+        groups_list.append(group['id'])
+
+    for group in groups_list:
+        users_in_group = json.loads(sc_client.get_users_of_groups(group))
+
+        for i in json_all_groups['groups']:
+            if i['id'] == group:
+                group_name = i['name']
```

Comment:
`group_name = json_all_groups['groups'][group]` ?
because groups_list is a complete extraction of json_all_groups['groups']
also it would be better to define a variable for `groups = json_all_groups['groups']` to make the code clearer